### DISCUSSION TITLE
More granular control over date/updated

### DISCUSSION THREAD

**Discussion by uncouthregions** [jpanther/congo#404] [Ideas]
I really like the addition of the Updated field. However, it looks like this only appears if a date parameter is set, and `showDate` is set to `true`.

I have some pages, such as a CV, where I want to show when they were updated, but not when they were first posted. Is it possible to do this?

I would also like to do so without disabling `showDate` entirely, as I want it to be visible on posts.

Thanks for the theme. It's amazingly elegant and easy to work with!

**Comment by jpanther** [comment]
This should get what you want...
```yml
showDate: false
showDateUpdated: true
```
